flocker/control/schema/endpoints.yml

```diff
@@ -28,6 +28,16 @@ definitions:
         '$ref': 'types.json#/definitions/environment'
       restart_policy:
         '$ref': 'types.json#/definitions/restart_policy'
+      volumes:
+        title: Volumes to attach to the container
+        description: |
+          Volumes to attach to the container. Required keys are "dataset_id" and
+          "mountpoint". For now only one volume per container is supported.
+        type: array
+        items:
+          '$ref': 'types.json#/definitions/volume'
+        minItems: 0
+        maxItems: 1
```

Comment:
:question: possibly easier to locate and maintain the detailed definitions if they're all in `types.yml` and `volumes` sticks to the same convention of just using a `$ref` here.